examples/Example1.cpp

```diff
@@ -24,11 +24,11 @@
 //
 //
 //  This piece of code was developed as part of ICE-DIP project at CERN.
-//  "ICE-DIP is a European Industrial Doctorate project funded by the European Community's 
+//  "ICE-DIP is a European Industrial Doctorate project funded by the European Community's
 //  7th Framework programme Marie Curie Actions under grant PITN-GA-2012-316596".
 //
 
-#include "../UMESimd.h"
```

Comment:
The main reason why I didn't move all files from main directory to any sub-directory is that these files are the best documentation available for the project so far. Hiding them from the users somewhere in the 'internals' will only result in decrease in readability and stepper learning curve.  I'm consulting files such as UMESimdinterface.h and UMEScalarEmulation.h on a regular basis myself, so having to dig through the hierarchy to get to them is additional burden.

Of course some of the files, such as UMEInline.h and UMEMemory.h are more like 'portability patches' than anything really helpful to the users, so they could be hidden somewhere.